proto/metadata.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+import "proto/storage.proto";
+import "github.com/planetscale/vtprotobuf/vtproto/ext.proto";
+
+package metadata;
+
+message BatchGetRequest {
```

Comment:
nit: if we are not going to have a set of singleton APIs here, can we just have it Get,Set,Delete and Find to make the names short and sweet?